css/styles.css

```diff
@@ -0,0 +1,194 @@
+@import url("http//fonts.googlepis.comcss?family=Lobster|Raleway:700,400,300");
+
+/* GLOBal */
+
+.container-fluid {
+
+    padding: 0;
+
+    margin: 0;
+
+}
+
+#OurStory, #Menu, #Review, #ContactUs {
+    min-height: 600px;
+} 
+.nav li a:hover {
+
+background-color: transparent; 
+    
+}
+
+.navbar-default {
+    
+    border-color: transparent;
+    background-color: transparent; 
+}
+
+.navbar-default .navbar-nav> .active > a,
+
+.navbar-default .navbar-nav> .active > a:hover,
+
+.navbar-default .navbar-nav> .active > a:focus {
+
+    color: #fff;
+    background-color: rgba(17, 17, 17, .5);
+
+}
+
+.navbar-default .navbar-nav > li > a {
+
+    color: #fcfcfc;
+    text-shadow: 1px 1px 2px #0a0a0a;
+}
+
+.navbar-default .navbar-nav li a:hover,
+.navbar-default .navbar-nav li a:focus {
+    color: #f8512f;
+    background-color: rgba(17, 17, 17, .2);
+}
+
+nav.navbar {
+    position: absolute; 
+    top: 100;
+    right: 0;
+    left: 0;
+    z-index: 20;
+}
+
+.navbar-brand > img {
+
+    margin-top: -7px;
+
+}
+
+.carousel {
+
+    margin-bottom: 60px;
+}
+
+.carousel-caption {
+    z-index: 10;
+    padding-bottom: 60px;
+
+}
+
+.carousel-inner .item {
+    width: 100%;
+    height: 600px;
+    background-size: cover;
+    background-position: center;
+}
+
+#carousel_image_one { 
+    
+    background-image: url(../img/wood-2.jpg); 
+
+}
+
+#carousel_image_two {
+
+    background-image: url(../img/restaurant-2.jpg); 
+
+}
+
+
+ #carousel_image_three {    
+
+    background-image: url(../img/farm-2.jpg);
+ }
+
+ .carousel_image_logo {
+
+    max-height: 300px;
+ }
+
+ .carousel_button {
+
+    color: #fff;
+    background-color: #d34326;
+    border-color: inherit;
+ }
+
+ .carousel_button:hover {
+     background-color: #d34326;
+     border-color: inherit
+
+ }
+
+ .carousel-capation p {
+     color: #f7f7f7;
+     font-size: 16px; 
+ }
+
+ /* OUR STORY */
+
+ #OurStory .row {
+
+margin-top: 30px;
+ }
+#our_story h2 {
+    font-family: 'lobster', cursive;
+    color: #f8512f;
+    font-size: 6rem;
+    text-shadow: 1px 1px 5px #0a0a0a;
+
+}
+
+
+#OurStory h3 {
```

Comment:
I think this should be `our_story`.

Reply:
Yes just changed it had to make a change in html as well.